includes/class-amp-theme-support.php

```diff
@@ -220,6 +220,14 @@ public static function add_hooks() {
 		remove_action( 'wp_print_styles', 'print_emoji_styles' );
 		remove_action( 'wp_head', 'wp_oembed_add_host_js' );
 
+		// Prevent MediaElement.js scripts/styles from being enqueued.
+		add_filter( 'wp_video_shortcode_library', function() {
+			return 'amp';
+		} );
+		add_filter( 'wp_audio_shortcode_library', function() {
+			return 'amp';
+		} );
+
```

Comment:
This change is here because I haven't yet merged #106 into `develop` from `0.7`.